apps/transformers/lib/transformations/arithmetic_subtract.ex

```diff
@@ -0,0 +1,71 @@
+defmodule Transformers.ArithmeticSubtract do
+  @behaviour Transformation
+
+  alias Transformers.Validations.IsPresent
+  alias Transformers.Validations.NotBlank
+  alias Transformers.Validations.ValidationStatus
+
+  @minuend "minuend"
+  @subtrahends "subtrahends"
+  @target_field "targetField"
+
+  @impl Transformation
+  def transform(payload, parameters) do
+    with {:ok, [minuend, subtrahends, target_field]} <- validate(parameters),
+         {:ok, difference} <- subtractValues(minuend, subtrahends, payload) do
+      {:ok, payload |> Map.put(target_field, difference)}
+    else
+      {:error, reason} -> {:error, reason}
+    end
+  end
+
+  defp subtractValues(minuend, subtrahends, payload) do
+    minuendValue = Map.get(payload, minuend, minuend)
+
+    if is_number(minuendValue) do
```

Comment:
This can likely be cleaned up if we have a validate_parameters and validate_payload in the future.